calico_containers/calicoctl.py

```diff
@@ -110,6 +110,6 @@
         print_paragraph(e.message)
         sys.exit(1)
     except BaseException as e:
-        print "Unexpected error executing command.\n"
+        print "{0}: {1}\n".format(type(e).__name__, e)
```

Comment:
Added this more to clarify in troubleshooting. If this isn't desired, I can certainly take it out.